app_server/app/Router.js

```diff
@@ -97,6 +97,9 @@ server.register(require('inert'), function(err) {
   server.route({
     method: 'GET',
     path: '/admin/{param*}',
+    config: {
+      auth: false
+    },
```

Comment:
Only the login "page" is accessible without logging in through normal usage.
As this is a single page application, the client decides which "page" to show.
There are no other actual pages, accessing other URLs directly will redirect them to the login; I'll get the login to show an error message and work on more detailed access control (different types of permission) later on, though.

The authentication applies on API requests, so while users are technically able to access the views (as it's publicly available), they cannot access the data.